x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/extractor/FieldHitExtractor.java

```diff
@@ -132,10 +131,6 @@ private Object unwrapMultiValue(Object values) {
             if (values instanceof String) {
                 return DateUtils.asDateTime(Long.parseLong(values.toString()));
             }
-            // returned by nested types...
```

Comment:
@costin I'm not sure about this, but doesn't seem to break something. Should we check and add tests?

Reply:
It can be removed now with Joda out. Likely this will eliminate `asDateTime` as well.